nhif-faq.html

```diff
@@ -0,0 +1,183 @@
+---
+layout: nhif-faq
+title: NHIF FAQ
+---
+<div class="container">
+  <div class="faq-container">
```

Comment:
Why did we add this div?

Reply:
@DavidLemayian I am sorry about that the faq-container div is not necessary I forgot to remove that when I was done.